tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -23,22 +23,21 @@ Login To RHODS Dashboard
    ${authorize_service_account} =  Is rhods-dashboard Service Account Authorization Required
    Run Keyword If  ${authorize_service_account}  Authorize rhods-dashboard service account
 
-Wait for RHODS Dashboard to Load
-  [Arguments]  ${dashboard_title}="Red Hat OpenShift Data Science Dashboard"
-  Wait For Condition  return document.title == ${dashboard_title}  timeout=15
+Wait for ${ods_project} Dashboard to Load
+  Wait For Condition  return document.title == "${${ods_project}_DASHBOARD_TITLE}"  timeout=15
 
-Wait Until RHODS Dashboard ${dashboard_app} Is Visible
+Wait Until Main Dashboard ${dashboard_app} Is Visible
   # Ideally the timeout would be an arg but Robot does not allow "normal" and "embedded" arguments
   # Setting timeout to 30seconds since anything beyond that should be flagged as a UI bug
   Wait Until Element is Visible  xpath://div[@class="pf-c-card__title" and .="${dashboard_app}"]  30seconds
 
 Launch ${dashboard_app} From RHODS Dashboard Link
```

Comment:
Shouldn't this be
`Launch ${dashboard_app} From ${ODS_PROJECT} Dashboard Link`
To keep consistent with the other changes?